src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -387,7 +410,8 @@ export function MapAnalysisImpl(props: Props & CompleteAppState) {
                 onAnalysisNameEdit={analysisState.setName}
                 onToggleExpand={() => setMapHeaderIsExpanded((c) => !c)}
                 studyName={studyRecord.displayName}
-                totalEntitesCount={totalEntityCount}
+                totalEntitesInSampleCount={totalEntitiesInSampleCount}
```

Comment:
For consistency I would name this back to the original minus the typo, so `totalEntitiesCount` 

This is the total unfiltered count of entities that comes from `totalCounts = useEntityCounts()`

To get the correct entity we need to capture the `outputEntity` return value from `useMapMarkers`

e.g. `totalCounts.value[outputEntity.id]` (or similar)